tasks/get-ocp-version/get-ocp-version.yaml

```diff
@@ -46,11 +49,6 @@ spec:
           # fetch the image base name containing the version for the found manifest image
           image_base_name=$(skopeo inspect --raw docker://"${manifest_image}" \
            | jq '.annotations."org.opencontainers.image.base.name"' | cut -d: -f2 | sed 's/"//g')
-
-        else
-          # fetch the image base name containing the version for a manifest image
-          image_base_name=$(skopeo inspect --raw docker://"${fbc_fragment}" \
```

Comment:
What I was saying is this is the same exact code as four lines above it, but it uses $fbc_fragment instead of manifest_image. So my hope was that `manifest_image="${fbc_fragment%@*}@${manifest_image_sha}"` can become `fbc_fragment="${fbc_fragment%@*}@${manifest_image_sha}"` and then this piece of code is still there (but not in an else). Then, you can remove
```
          # fetch the image base name containing the version for the found manifest image
          image_base_name=$(skopeo inspect --raw docker://"${manifest_image}" \
           | jq '.annotations."org.opencontainers.image.base.name"' | cut -d: -f2 | sed 's/"//g')
```
from the if

Reply:
ah ok. Now I got you. In my mind for some reason I thought that eventually the image could have a different prefix - which will never be the case. Thanks for the clarification, Johnny.

Reply:
Actually, I think (with some adjusts) in my proposal we can save one skopeo call - we only call it the 2nd time if the image is multi-arch.

Reply:
Sorry, I was on PTO and it is taking me time to catch up on notifications. This looks great to me now. Little surprised the linter isn't complaining that like `media_type=$(jq -r .mediaType <<< "${image_metadata}")` isn't quoted, but :man_shrugging: if the linter likes it, all good

Reply:
Yeah, as I said elsewhere today, apparently it doesn't require "" if you're assigning to a variable.

Chatgpt said this when I asked why:

> This behavior is because Bash performs word splitting and globbing (expansion of wildcard characters) in certain contexts, but not when assigning to a variable.

I think that's a plausible explanation.

It also doesn't expand this for example:
```
$ myvar=tmp/*
$ echo $myvar
tmp/*
```